web/src/hashchange.js

```diff
@@ -323,6 +343,12 @@ function do_hashchange_overlay(old_hash) {
         if (base === "settings") {
             settings_panel_menu.normal_settings.activate_section_or_default(section);
         } else {
+            let right_section;
+            if (section === "user-list-admin") {
+                const current_right_section = hash_parser.get_current_nth_hash_section(2);
+                right_section = handle_invalid_users_section_url(current_right_section);
+                settings_panel_menu.org_settings.curr_right_section = right_section;
+            }
             settings_panel_menu.org_settings.activate_section_or_default(section);
```

Comment:
Here the `activate_section_or_default` call can be removed and the `curr_right_section` field should be set.